### PR TITLE
E2E test for SDK’s WASM wrapper

### PR DESCRIPTION
**Pull request by sekulicd**
This contains WASM end to end test, it is using playwright-go which is used to enable cross-browser web automation.
Basically i created minimal index.html that just loads wasm. I serve it with wasm_exec.js and ark-sdk.wasm and then i simulate Alice sends vtxos to Bob using headless browsing.
This is usefull as with this we test not only WASM but SDK as well.
GH action added.

Closing prev one as GHA seems cant be triggered from fork.

@tiero @altafan @louisinger @bordalix please review.

### CODE REVIEW

**Comment by louisinger** [comment]
Running the test locally, it fails with:
```
 playwright: Error: Post "http://localhost:7070/v1/round/tree/submitNonces": net/http: fetch() failed: TypeError: Failed to fetch
```

**Comment by sekulicd** [comment]
> Running the test locally, it fails with:
> 
> ```
>  playwright: Error: Post "http://localhost:7070/v1/round/tree/submitNonces": net/http: fetch() failed: TypeError: Failed to fetch
> ```

communicated offline, after rebuilding docker img from branch it worked

**Review by louisinger — APPROVED** [review]
tACK

**Comment by sekulicd** [comment]
@altafan please review.

File: .github/workflows/ark.unit.yaml [hunk 1]
```diff
@@ -78,10 +78,7 @@ jobs:
         with:
           version: v1.61
           working-directory: ./pkg/client-sdk
-          config-content: |
-            issues:
-              exclude-files:
-                - '^test/wasm/wasm_test\.go$'
+          args: --timeout 5m --skip-files=.*_test.go
```
Comment (altafan) [inline .github/workflows/ark.unit.yaml:81]: `--skip-files` not needed, can be dropped?

Comment (sekulicd) [reply .github/workflows/ark.unit.yaml:81]: I am having linting errors in wasm_test.go also I was thinking that linting tests doesnt make much sense if tests are passing, we only lose bit time in CI and bit more when inspecting linting errors.

**Comment by tiero** [comment]
Better and more descriptive title (hence better commit message when merged)

**Comment by sekulicd** [comment]
> Better and more descriptive title (hence better commit message when merged)

This one ok?

**Pull request merged**